src/transformation/visitors/break-continue.ts

```diff
@@ -8,7 +8,8 @@ import { findScope, ScopeType } from "../utils/scope";
 export const transformBreakStatement: FunctionVisitor<ts.BreakStatement> = (breakStatement, context) => {
     const breakableScope = findScope(context, ScopeType.Loop | ScopeType.Switch);
     if (breakableScope?.type === ScopeType.Switch) {
-        return lua.createGotoStatement(`____switch${breakableScope.id}_end`);
+        // Break is handled by the switch statement (see transformSwitchStatement)
+        return lua.createBreakStatement(breakStatement);
```

Comment:
Isn't this redundant?

Reply:
True. was leaving it there for a smaller delta. If I remove everything else out of that function, the context is no longer needed, guessing I should just do a `void context;` to silence the unused error?